### DISCUSSION TITLE
Ordered list with numbers inside of unordered list

### DISCUSSION THREAD

**Discussion by TheCataliasTNT2k** [squidfunk/mkdocs-material#7006] [Q&A] [answered]
Hey, I want to have an ordered list with numbers inside an unordered list.

```
- Test
- Test 2
- Test 3
    1. First item
    2. Second item
- Test 4
```

But even here on GitHub, the numbers are replaced by "i." and "ii.". How can I keep the numbers?
Mkdocs replaces them by "a" and "b" and so forth.
- Test
- Test 2
- Test 3
    1. First item
    2. Second item
- Test 4

(To clarify: The ordered list is not nested inside another ordered list, like shown above)

**Comment by facelessuser** [comment]
Python Markdown requires you to indent 4 spaces.

**Reply by TheCataliasTNT2k** [reply]
Yes, I know, this was just an example, which works in GitHub.
But four spaces can be rendered by GitHub the same.

**Reply by facelessuser** [reply, marked as answer]
Oh, I see. I guess things were a bit unclear to me. You can force numbering styles with CSS.

```css
ol > li {
    list-style-type: decimal;
}
```

**Reply by TheCataliasTNT2k** [reply]
Do you have a link, how to add custom CSS? Would be great ;)

**Reply by facelessuser** [reply]
https://squidfunk.github.io/mkdocs-material/customization/?h=custom#additional-css

**Reply by TheCataliasTNT2k** [reply]
Thank you!

**Reply by kamilkrzyskow** [reply]
Yep, the same solution was suggested here:
- https://github.com/squidfunk/mkdocs-material/discussions/6909#discussioncomment-8788221

**Comment by squidfunk** [comment]
After thinking a little about this, I believe we need to come back to this problem and fix it. An ordered list inside an unordered lists should not receive roman lowercase alphanumerical numbering, it should be numbers, i.e., 1., 2., 3.
